util/contentutil/copy.go

```diff
@@ -46,6 +46,18 @@ func (r *rc) Read(b []byte) (int, error) {
 	return n, err
 }
 
+func (r *rc) Seek(offset int64, whence int) (int64, error) {
+	switch whence {
+	case io.SeekStart:
+		r.offset = int(offset)
```

Comment:
Not really related to this PR, but it seems like it would be better for `offset` to be an `int64`.